jcliStart.go

```diff
@@ -45,45 +51,44 @@ func main() {
 		input = strings.TrimSpace(input)
 
 		if strings.ToLower(input) == "login" {
-			login()
+			err = login()
+
+			printError(err)
```

Comment:
I wonder if a function name like `handleError` or `printIfErrored` would make more sense here in order to make it clear that you are not necessarily in the error path.

Also, since each "command" now returns an error type, you could just do the error handling once at the end of the command parsing conditional.